cmd/neofs-node/netmap.go

```diff
@@ -182,9 +184,26 @@ func initNetmapService(c *cfg) {
 	addNewEpochAsyncNotificationHandler(c, func(ev event.Event) {
 		e := ev.(netmapEvent.NewEpoch).EpochNumber()
 
-		ni, err := c.netmapLocalNodeState(e)
+		var (
+			ni  *netmapSDK.NodeInfo
+			err error
+		)
+
+		expBackoff := backoff.NewExponentialBackOff()
+		expBackoff.InitialInterval = 200 * time.Millisecond
+		expBackoff.MaxElapsedTime = 5 * time.Second
+
+		err = backoff.RetryNotify(
+			func() error {
+				ni, err = c.netmapLocalNodeState(e)
+				return err
+			},
+			expBackoff,
+			func(err error, d time.Duration) {
+				c.log.Warn("retrying due to error", zap.Uint64("epoch", e), zap.Error(err), zap.Duration("retry-after", d))
```

Comment:
Info?

Reply:
why? i does not feel it info at all

Reply:
It's a pretty normal thing. Node is busy, ok, retrying. Now if we have no success after N retries then it's problem, but no earlier than that.